mitodl/ocw-hugo-themes
# more search css changes

#### Pre-Flight checklist

<img width="1673" alt="Screen Shot 2022-03-23 at 12 29 35 AM" src="https://user-images.githubusercontent.com/1934992/159624270-7aaf7ed3-83fb-41af-ac97-503ac35bd3c1.png">

![Screen Shot 2022-03-23 at 12 01 17 AM](https://user-images.githubusercontent.com/1934992/159621699-129faffb-67bf-4f87-8831-1b568635ebc1.png)
<img width="1673" alt="Screen Shot 2022-03-23 at 12 30 35 AM" src="https://user-images.githubusercontent.com/1934992/159624377-42ebba41-c099-40eb-8d93-27edb58b0648.png">
![Screen Shot 2022-03-23 at 12 02 59 AM](https://user-images.githubusercontent.com/1934992/159621710-9b67a414-9bdd-4303-a8e9-b97b299d12aa.png)
![Screen Shot 2022-03-23 at 12 02 45 AM](https://user-images.githubusercontent.com/1934992/159621711-4b7f82b6-8b7f-44d5-b3a4-f285d55ad144.png)

#### What are the relevant tickets?
None

#### What's this PR do?
This PR improves the mobile UI for the course and research search and makes tweaks to the desktop UI requested by Ferdi, Daniel and Nicolae. The changes are as follows:

1) Change the resource search UI to look like the v1.0 design from the v1.1 design. The design is here:
https://projects.invisionapp.com/d/main?origin=v7#/console/21537330/465125995/preview?scrollOffset=0
2) Even out the gray blocks on either side of the course cards
3) Make course titles bold
4) Make “+x more” for courses and instructors into links
5) A few additional css updates suggested by Nicolae
6) Improve the mobile ui for resource and course search. The mobile designs are 
https://projects.invisionapp.com/d/main?origin=v7#/console/21537330/465292558/inspect?scrollOffset=0 and 
https://projects.invisionapp.com/d/main?origin=v7#/console/21537330/465252649/inspect?scrollOffset=0

 These designs arn't finalized so the UI doesn't look exactly like the designs. In particular it wasn't super clear to me how the mobile resource search ui should look with videos. I displayed just the play icon for the resource type icon for now since it's the same size as the file/pdf/html size icons but it definitely makes sense to replace it with a custom mobile sized video resource icon.

#### How should this be manually tested?
Run the course and resource search UIs locally. Verify that things look good at different screen sizes

The test error is not from this commit
